src/data/schema.rs

```diff
@@ -22,13 +31,39 @@ impl ColumnDefExt for ColumnDef {
             .any(|ColumnOptionDef { option, .. }| option == &ColumnOption::Null)
     }
 
+    #[cfg(feature = "auto-increment")]
+    fn is_auto_incremented(&self) -> bool {
```

Comment:
Good to see this in `ColumnDefExt` :)
However I saw there are still a few redundant codes in other files. You might be able to replace them to use this function.